docs/.docz/doczrc.js

```diff
@@ -0,0 +1,36 @@
+// https://www.docz.site/docs/project-configuration
```

Comment:
What is this file used for now?  It doesn't seem to be up to date with what I'm viewing on the site.

Reply:
Ok, per docz own documentation we should gitignore the whole `.docz` folder:

https://github.com/doczjs/docz/blob/master/.gitignore#L34